crates/ruff_linter/src/rules/flake8_pyi/snapshots/ruff_linter__rules__flake8_pyi__tests__PYI059_PYI059.py.snap

```diff
@@ -0,0 +1,74 @@
+---
+source: crates/ruff_linter/src/rules/flake8_pyi/mod.rs
+---
+PYI059.py:8:7: PYI059 [*] `Generic[]` should always be the last base class
+   |
+ 6 | V = TypeVar('V')
+ 7 | 
+ 8 | class LinkedList(Generic[T], Sized):  # PYI059
+   |       ^^^^^^^^^^ PYI059
```

Comment:
I think it would be better if the carets underlined the bases tuple here, rather than the name of the class, since that's where the error is. That means we need to pass the range of the bases tuple in as the second argument of `Diagnostic::new()` rather than `class_def.identifier()`